### PR TITLE
Add generic game styling; Add game components stubs;

### PR DESCRIPTION
**Pull request by kirilknysh**

### CODE REVIEW

File: public/common/stylesheets/color-palette.styl [hunk 1]
```diff
@@ -1,5 +1,7 @@
 /* Palette */
+black-velours = #212121 //dark, mainly background
 
-light-green  = #a7ffa7
-light-red    = #ff9b9b
-light-orange = #ffc469
+fresh-dollar = #349161 //main green colour
+old-dollar = #1a4931 //dark green colour
+
+nectarine = #FF3300 //the color of fail
```
Comment (DaQuirm) [inline public/common/stylesheets/color-palette.styl]: "the color of despair" :smile_cat:

Comment (DaQuirm) [inline public/common/stylesheets/color-palette.styl]: please, keep the case consistent for color values :cat:

File: public/game/styles/game.styl [hunk 2]
```diff
@@ -0,0 +1,48 @@
+.game-screen
+	.user-panel
+		width: 100%
```
Comment (DaQuirm) [inline public/game/styles/game.styl]: No colons in `.styl`, right?

File: public/game/styles/game.styl [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+		box-sizing: border-box
+
+@media (orientation: portrait)
+	.game-screen-container
+		width: 96%
+
+//NON-PRODUCTION styles; only for testing
+.user-panel
+	background: #888
```
Comment (DaQuirm) [inline public/game/styles/game.styl]: Is there something wrong with indentation here?..

**Comment by DaQuirm** [comment]
Apart from the nitpicks, LGTM
Trying it out locally now

File: public/common/stylesheets/color-palette.styl (hunk 1 above)
Comment (kirilknysh) [inline public/common/stylesheets/color-palette.styl]: you mean camelCase?

File: public/game/styles/game.styl (hunk 3 above)
Comment (kirilknysh) [inline public/game/styles/game.styl]: mmm, no. it should be so. Everything below the comment will be removed in the nearest future.

File: public/common/stylesheets/color-palette.styl (hunk 1 above)
Comment (DaQuirm) [inline public/common/stylesheets/color-palette.styl]: no, just like `#ff3300`

Comment (kirilknysh) [inline public/common/stylesheets/color-palette.styl]: ah, of course! copy-paste-ing :(

**Pull request merged**